python/fastexcel/_fastexcel.pyi

```diff
@@ -18,10 +21,48 @@ class _ExcelSheet:
 class _ExcelReader:
     """A class representing an open Excel file and allowing to read its sheets"""
 
-    def load_sheet_by_name(self, name: str) -> _ExcelSheet:
-        """Loads a sheet by name"""
-    def load_sheet_by_idx(self, idx: int) -> _ExcelSheet:
-        """Loads a sheet by index"""
+    def load_sheet_by_name(
+        self,
+        name: str,
+        /,
+        header_row: int | None = 0,
+        column_names: list[str] | None = None,
+    ) -> _ExcelSheet:
+        """Loads a sheet by name.
+
+        The header_row (row used for the column labels) is by default the first row.
+        but another value for the header line can be specified.
+        If `None` is passed it means that the sheet doesn't have any header.
+        Column names can override headers found in the document.
+        If column_names is used header_row will be ignored."""
+    def load_sheet_by_idx(
+        self,
+        idx: int,
+        /,
+        header_row: int | None = 0,
+        column_names: list[str] | None = None,
+    ) -> _ExcelSheet:
+        """Loads a sheet by his index.
+
+        The header_row (row used for the column labels) is by default the first row.
+        but another value for the header line can be specified.
+        If `None` is passed it means that the sheet doesn't have any header.
+        Column names can override headers found in the document.
+        If column_names is used header_row will be ignored."""
```

Comment:
All these docstrings are nice but they need to be moved to the proxy classes in `__init__.py`, otherwise they won't be accessible to users

Reply:
should we just `pass` than the methods in `_fastexcel.pyi` ?

Reply:
I'd leave a dosctring containing only the first line. It doesn't take more LoC than `pass` but still gives a quick recap. No strong opinions though, whatever you prefer. (In case of type stubs, I've almost always seen `...` rather than `pass`)